pkg/tfbridge/schema.go

```diff
@@ -190,6 +189,10 @@ const defaultsKey = "__defaults"
 // marshaled back to assets by MakeTerraformOutputs.
 type AssetTable map[*SchemaInfo]resource.PropertyValue
 
+// ErrSchemaDefaultValue is used internally to avoid a panic in pf/schemashim.DefaultValue().
+// See https://github.com/pulumi/pulumi-cloudflare/issues/460
```

Comment:
Nit, this is not a great issue to link into the source code for historic reference, as it does not explain the end to-end context of the problem. Let me write an issue here in terraform-bridge and we can link it instead.